packages/now-routing-utils/src/superstatic.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * This converts Superstatic configuration to Now.json Routes
+ * See https://github.com/firebase/superstatic#configuration
+ */
+
+import pathToRegexp from 'path-to-regexp';
+import { Route, NowRedirect, NowRewrite, NowHeader } from './types';
+
+export function convertCleanUrls(filePaths: string[]): Route[] {
+  const htmlFiles = filePaths
+    .map(toRoute)
+    .filter(f => f.endsWith('.html'))
+    .map(f => ({
+      html: f,
+      clean: f.slice(0, -5),
+    }));
+
+  const redirects: Route[] = htmlFiles.map(o => ({
+    src: o.html,
+    headers: { Location: o.clean },
+    status: 301,
+  }));
+
+  const rewrites: Route[] = htmlFiles.map(o => ({
+    src: o.clean,
+    dest: o.html,
```

Comment:
I suspect this won't work without `contentType`, I think the solution here might be on the builder side of things

Reply:
Why won't it work? Can you provide an example that would not work?

We can't put this in every builder, it must work for all builders.

Reply:
I think I'm thinking of an issue I saw when implementing builder routes, it might not be related.

What I mean by on the builder side of things is I think we should remove `.html` from the file entry itself and add the right `contentType` so it gets picked up in `{ "handle": "filesystem" }` instead of after it.

Reply:
I think I'll have to pass a parameter to `detectRoutes(cleanUrls)` which will skip the zero config route logic and do it all here instead.